opencti-platform/opencti-graphql/src/manager/historyManager.ts

```diff
@@ -117,6 +118,7 @@ const eventsApplyHandler = async (context: AuthContext, events: Array<SseEvent<S
       applicant_id: event.data.origin?.applicant_id,
       timestamp: eventDate,
       context_data: contextData,
+      authorized_members: stix.extensions[STIX_EXT_OCTI].authorized_members,
```

Comment:
why do we want to save authorized members in history index ?

Reply:
To keep same behavior as for marking and organization sharing systems